components/edit-collective/sections/Tiers.js

```diff
@@ -165,6 +167,42 @@ const Tiers = ({ collective }) => {
                 )}
               </Mutation>
             </Box>
+            {cryptoContributionsEnabledByHost && (
+              <Box mb={4}>
+                <StyledHr my={4} borderColor="black.300" />
+                <P fontSize="14px" lineHeight="20x" mb={3}>
+                  <FormattedMessage defaultMessage="Enabling this will add a tier that allows users to support your collective with Cryptocurrencies such as Bitcoin or Ethereum." />
```

Comment:
```suggestion
                  <FormattedMessage
                            id="Tiers.CryptoTierDescription"
                            defaultMessage="Enabling this will enable support for donations with Cryptocurrencies such as Bitcoin or Ethereum."
                          />
```

Reply:
Why re-invent a new string when we could just re-use the one we had before?

Reply:
I thought the style of language of this is different that in the previous string `The custom contribution adds a default tier on your collective that doesn't enforce any minimum amount or interval. This is the easiest way for people to contribute to your Collective, but it cannot be customized.` and hence changed it to mimic that. But I am okay either way so reverted this as you suggested.